R/install.snowball.R

```diff
@@ -72,69 +72,130 @@ install.snowball <- function(pkg, date, include.suggests, force.install = FALSE,
 
       # 3.5 INSTALL K FROM CRAN
       if (from.k == "CRAN") {
+        message1("\n>Installing Binary from CRAN")
         install.packages(pkg.k, dependencies = FALSE, lib = lib.k, repos = "https://cloud.r-project.org", type = "both", quiet = quiet.install, keep_outputs = file.path(get.groundhog.folder(), "logs/"))
-        message1("Installing Binary from CRAN")
       }
 
       # 3.6 INSTALL K FROM MRAN
       if (from.k == "MRAN") {
-        # Try MRAN 1
+        message1("\n>Attempting to install binary from MRAN\n")
```

Comment:
Messages already include new lines by default. You can check this with `message("test1"); message("test2")`. Do you want an additional blank line?